docker-compose.yml

```diff
@@ -115,6 +115,8 @@ services:
     build:
       context: ./
       dockerfile: io-functions-services/Dockerfile
+      args:
+        - FUNCTIONS_SERVICES_BRANCH
```

Comment:
probably needs to be added to every other function as well?

Reply:
fixed in fd5ec2b